extensions/ql-vscode/src/query-history/query-history-manager.ts

```diff
@@ -1547,9 +1476,11 @@ the file in the file explorer and dragging it into the workspace.`,
         false,
       );
     } else if (item.t === "remote") {
-      await this.remoteQueriesManager.openRemoteQueryResults(item.queryId);
+      // Do nothing. TODO: Remove when remote queries is no longer supported.
     } else if (item.t === "variant-analysis") {
       await this.variantAnalysisManager.showView(item.variantAnalysis.id);
+    } else {
+      assertNever(item);
```

Comment:
We're not removing the branch for checking whether an item is `remote`. Why do we need to add the extra `else` here?

Reply:
We don't necessarily need to, but this makes it consistent with most of the other `if-else` constructs. This will make it an error if we add a new possible `item.t` value. I'm happy to remove it, but I do think it adds some extra safety.

Reply:
I see! In which case it sounds like a good addition. Thanks for explaining.